js-jasmine/src/agedbrie.js

```diff
@@ -0,0 +1,19 @@
+
+class AgedBrie {
```

Comment:
I like how you've broken each item into it's own class, very concise and easy to read :)